### ISSUE TITLE
feat(password): check strength

### ISSUE THREAD

**Issue by dargmuesli** [maevsi/maevsi#747]
Use https://yarnpkg.com/package/tai-password-strength server side.
Idea by @azolus

**Comment by vibh1103** [comment]
Hi @dargmuesli  @PatrickMi can i work on this task ?

**Comment by dargmuesli** [comment]
@vibh1103 yes, you're welcome. Please feel free to ask any questions! You can join the Discord linked at https://github.com/maevsi to reach us.

**Comment by dargmuesli** [comment]
@vibh1103 are you still working on this?
